src/Sulu/Bundle/MediaBundle/Tests/Functional/Media/Storage/S3StorageTest.php

```diff
@@ -19,14 +19,9 @@
 
 class S3StorageTest extends SuluTestCase
 {
-    /**
-     * @var string
-     */
-    protected static $class = S3Kernel::class;
-
-    protected static function getKernelClass(): string
+    protected function setUp(): void
     {
-        return S3Kernel::class;
+        static::$class = S3Kernel::class;
```

Comment:
Are you missing a call to `parent::setUp()`?

Reply:
there is no parent setUp functionality but setting the static values has effects to the next class so we need in teardown reset it.